api/agent/config.go

```diff
@@ -0,0 +1,60 @@
+package agent
+
+import (
+	"errors"
+	"math"
+	"os"
+	"strconv"
+	"time"
+)
+
+type AgentConfig struct {
+	MinDockerVersion string        `json:"minDockerVersion"`
+	FreezeIdleMsecs  time.Duration `json:"freezeIdleMsecs"`
+	EjectIdleMsecs   time.Duration `json:"ejectIdleMsecs"`
```

Comment:
while we're here, by convention these should be `min_docker_version`, etc. -- if we ever have it so simple as being able to use a json config

Reply:
lost specificity: `json:"min_docker_version"` - the go fields themselves being camel cased is appropriate